src/form/Form.js

```diff
@@ -75,6 +82,24 @@
                  context.click();
              })
         ;
+
+    };
+
+    Form.prototype._validate = function(context) {
+        return this.columns().every(function(d, i) {
+            var valStr = d.validate;
```

Comment:
(I need to test that i dont need a "typeof" check) but it should just return true .. that was the goal atleast but i will relook over before any real PR